pyaedt/edb_core/layout.py

```diff
@@ -59,26 +59,25 @@ def db(self):
 
     @property
     def layers(self):
-        """
+        """Retrieve layers.
 
         Returns
         -------
         dict
-            Dictionary of Layers
+            Dictionary of layers.
         """
         return self._parent.core_stackup.stackup_layers.layers
 
     @aedt_exception_handler
     def update_primitives(self):
         """
```

Comment:
Should be on the same line as the start of the docstring.

Reply:
@akaszynski  I don't understand what you mean by this comment.  What should be on the same line as the start of the docstring?